qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -106,14 +106,44 @@ unmap_device()
     local dev=$1
     local pid=$2
 
-    _sudo rbd-nbd unmap ${dev}
-    rbd-nbd list-mapped | expect_false grep "^${pid}\\b" || return 1
+    _sudo rbd device --device-type nbd unmap ${dev}
+    rbd device --device-type nbd list | expect_false grep "^${pid}\\b" || return 1
     ps -C rbd-nbd | expect_false grep "^ *${pid}\\b" || return 1
 
     # workaround possible race between unmap and following map
     sleep 0.5
 }
 
+function test_default_pool()
+{
+    local pool=new_default_pool
+    local img=foo
+
+    ceph osd pool delete ${pool} ${pool} --yes-i-really-really-mean-it || true
+    ceph osd pool create ${pool} 32
```

Comment:
Move creation to setup() and deletion to cleanup(), so that if something goes wrong during the default pool test case, the pool is cleaned up.